packages/core/src/Account.ts

```diff
@@ -313,24 +323,28 @@ export class Account<T = any> extends EventEmitter {
    * Will try to get the load the local client from local DB.
    * If clientInfo are provided, will also create the client on backend and DB
    * If clientInfo are not provided, the method will fail if local client cannot be found
+   * If dbMigrationHooks are provided, and store name was found, will trigger migration script
    *
    * @param loginData User's credentials
    * @param clientInfo Will allow creating the client if the local client cannot be found (else will fail if local client is not found)
    * @param entropyData Additional entropy data
+   * @param dbMigrationHooks Cryptobox to CoreCrypto migration config object
    * @returns The local existing client or newly created client
    */
   public async initClient(
     loginData: LoginData,
     clientInfo?: ClientInfo,
     entropyData?: Uint8Array,
+    dbMigrationHooks?: DBMigrationHooks,
```

Comment:
Since all those are optional, let's gather them in a `option` object param (this will avoid the `initClient(info, undefined, undefined, migrationHooks)`)